xml/System.Numerics/Complex.xml

```diff
@@ -1858,11 +1897,12 @@
         <Parameter Name="right" Type="System.Numerics.Complex" Index="1" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="left">To be added.</param>
-        <param name="right">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="left">The double value to add.</param>
```

Comment:
It might be good to keep both parts? `The double-precision real value to add.`

I do think it would be good to call out that this is the `real` value.

Reply:
Great, thank you. I'll commit this:

```suggestion
        <param name="left">The double-precision real value to add.</param>
```